src/tooling/espflash.md

```diff
@@ -1,20 +1,22 @@
 # espflash
 
-`espflash` provides a serial flasher utility for Espressif SoCs and modules based on [esptool.py].
+`espflash` is a serial flasher utility, based on [esptool.py], for Espressif SoCs and modules.
+
+The [esp-rs/espflash] repository contains two crates, `cargo-espflash` and `espflash`. For more information on these crates, see the respective sections below.
 
-The [esp-rs/espflash] repository contains two crates, `cargo-espflash` and `espflash`. You can find more information on both of these in their respective sections below and in their corresponding README.
 
 > #### A note on `espflash` and `cargo-espflash`.
 >
-> The `espflash` and `cargo-espflash` commands shown below, assume that version `2.0` or greater.
+> The `espflash` and `cargo-espflash` commands shown below, assume that version `2.0` or greater is used.
 
 [esp-rs/espflash]: https://github.com/esp-rs/espflash
 [esptool.py]: https://github.com/espressif/esptool
+
 ## cargo-espflash
 
 Provides a subcommand for `cargo` that handles cross-compilation and flashing.
```

Comment:
_Provides_ a subcommand or _is_ a subcommand?

Reply:
In this case, `espflash` is a subcommand for `cargo`, so I think we are right saying that it provide a subcommand.